tests/e2e/lib/velero_helpers.go

```diff
@@ -0,0 +1,158 @@
+package lib
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	snapshotv1beta1api "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1beta1"
+	snapshotv1beta1client "github.com/kubernetes-csi/external-snapshotter/client/v4/clientset/versioned"
+	velero "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	pkgbackup "github.com/vmware-tanzu/velero/pkg/backup"
+	"github.com/vmware-tanzu/velero/pkg/cmd"
+	"github.com/vmware-tanzu/velero/pkg/cmd/util/downloadrequest"
+	"github.com/vmware-tanzu/velero/pkg/cmd/util/output"
+	"github.com/vmware-tanzu/velero/pkg/features"
+	veleroClientset "github.com/vmware-tanzu/velero/pkg/generated/clientset/versioned"
+	"github.com/vmware-tanzu/velero/pkg/label"
+	"github.com/vmware-tanzu/velero/pkg/restic"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetVeleroClient() (veleroClientset.Interface, error) {
+	if vc, err := veleroClientset.NewForConfig(getKubeConfig()); err == nil {
+		return vc, nil
+	} else {
+		return nil, err
+	}
+}
+
+// https://github.com/vmware-tanzu/velero/blob/11bfe82342c9f54c63f40d3e97313ce763b446f2/pkg/cmd/cli/backup/describe.go#L77-L111
+func DescribeBackup(ocClient client.Client, backup velero.Backup) string {
+	err := ocClient.Get(context.Background(), client.ObjectKey{
+		Namespace: backup.Namespace,
+		Name:      backup.Name,
+	}, &backup)
+	if err != nil {
+		return "could not get provided backup: " + err.Error()
+	}
+	veleroClient, err := GetVeleroClient()
+	if err != nil {
+		return err.Error()
+	}
+	details := true
+	insecureSkipTLSVerify := true
+	caCertFile := ""
+
+	deleteRequestListOptions := pkgbackup.NewDeleteBackupRequestListOptions(backup.Name, string(backup.UID))
+	deleteRequestList, err := veleroClient.VeleroV1().DeleteBackupRequests(backup.Namespace).List(context.TODO(), deleteRequestListOptions)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error getting DeleteBackupRequests for backup %s: %v\n", backup.Name, err)
```

Comment:
I think we use `log.Printf` for printing errors, outputs, comments, it would be better if we can use the same here.